aries-site/src/pages/whats-new.mdx

```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## August 30 - September 3, 2021
+
+- Published [Notifications page](https://design-system.hpe.design/components/notification) 
+show the first variant of using [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications)
```

Comment:
Either link to the "first variant" using "/components/notification#toast-notification" or separate.

```suggestion
- Published [Notifications page](/components/notification).
- Published [Toast Notifications](/templates/toast-notifications).
```

```suggestion
- Published [Notifications page](/components/notification) 
showing the [first variant](/components/notification#toast-notification) using [Toast Notifications](/templates/toast-notifications).
```